spec/swagger_schemas/models/application.rb

```diff
@@ -224,249 +191,13 @@
     },
   },
 }.tap { |h|
-  h[:v2] = h[:v1]
-  h[:v3] = {
-    description: "A single NPQ application",
-    type: :object,
-    required: %i[id type attributes],
-    properties: {
-      id: {
-        "$ref": "#/components/schemas/IDAttribute",
-      },
-      type: {
-        description: "The data type",
-        type: :string,
-        example: "npq_application",
-        enum: %w[
-          npq_application
-        ],
-      },
-      attributes: {
-        properties: {
-          participant_id: {
```

Comment:
I think mainly this was done to maintain the order of the fields in the API to match what we surface for that version, losing this will mean that the order in the schema is affected 🤔 happy to chat about this